charts/weave-gitops/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v2
+name: weave-gitops
+description: A Helm chart for Kubernetes
```

Comment:
Should this be "A helm chart for Weave GitOps"

Reply:
Oh yeah good point.